internal/mode/static/telemetry/collector.go

```diff
@@ -190,6 +200,27 @@ func collectGraphResourceCount(
 
 	ngfResourceCounts.BackendTLSPolicyCount = int64(len(g.BackendTLSPolicies))
 
+	for policyKey, policy := range g.NGFPolicies {
+		switch gvk := policyKey.GVK; gvk.Kind {
```

Comment:
why not just `switch policyKey.GVK.Kind` ?

Reply:
Ah still getting used to these switch statements, originally I was switching on something else and I changed it to this but I guess I just missed this easy refactor, thanks!